examples/experimental/bitmap/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
```

Comment:
Could we use `html-webpack-plugin` to get rid of this boilerplace `index.html` file? Other examples were recently updated to remove `index.html`